Solutions/Menes.Abstractions/Menes/Internal/OpenApiLinkOperationMapper.cs

```diff
@@ -81,14 +81,14 @@ private static string GetKey(string context, MediaType mediaType)
 
         private static string GetContentType<T>()
         {
-            ContentFactory.TryGetContentType<T>(out string contentType);
+            ContentFactory.TryGetContentType<T>(out string? contentType);
             return contentType ?? typeof(T).FullName!;
         }
 
         private bool TryGetOperationId(string contentType, string relationType, string context, out string operationName)
         {
             var targetMediaType = new MediaType(contentType, relationType);
-            return this.operationMappings.TryGetRecursive(targetMediaType, m => GetKey(context, m), out operationName);
+            return this.operationMappings.TryGetRecursive(targetMediaType, m => GetKey(context, m), out operationName!);
```

Comment:
This is another case where with correct attributes on the method being called, you shouldn't need the `!`.